src/IssueCreator/Logging/FileLogger.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IssueCreator.Logging
+{
+    public class FileLogger
+    {
+        private string _logPath;
+        public FileLogger(string logPath = "")
+        {
+            if (string.IsNullOrEmpty(logPath))
+            {
+                logPath = "log.txt";
+            }
+
+            _logPath = logPath;
+        }
+
+        public void Log(string message)
+        {
+            lock (_logPath)
+            {
+                File.AppendAllText(_logPath, $"{DateTime.Now.ToShortDateString()} {DateTime.Now.ToShortTimeString()}: {message} {Environment.NewLine}");
```

Comment:
Time would be more important. Something fixed-width, as well, so scanning logs is easier. Perhaps the "s" format for ISO8601 sans UTC offset (would use local time, which is usually better for self-diagnosis of client app issues, compared with diagnosing global services).